rules/windows/discovery_privileged_localgroup_membership.toml

```diff
@@ -112,75 +111,46 @@ If enabling an EQL rule on a non-elastic-agent index (such as beats) for version
+        "Tactic: Discovery",
+        "Resources: Investigation Guide"
+        ]
 timestamp_override = "event.ingested"
-type = "eql"
+type = "new_terms"
 
 query = '''
-iam where event.action == "user-member-enumerated" and
-
-  /* excluding machine account */
-  not winlog.event_data.SubjectUserName: ("*$", "LOCAL SERVICE", "NETWORK SERVICE") and
-
-  /* noisy and usual legit processes excluded */
-  not winlog.event_data.CallerProcessName:
-               ("-",
-                "?:\\Windows\\System32\\VSSVC.exe",
-                "?:\\Windows\\System32\\SearchIndexer.exe",
-                "?:\\Windows\\System32\\CompatTelRunner.exe",
-                "?:\\Windows\\System32\\oobe\\msoobe.exe",
-                "?:\\Windows\\System32\\net1.exe",
-                "?:\\Windows\\System32\\svchost.exe",
-                "?:\\Windows\\System32\\Netplwiz.exe",
-                "?:\\Windows\\System32\\msiexec.exe",
-                "?:\\Windows\\SysWOW64\\msiexec.exe",
-                "?:\\Windows\\System32\\CloudExperienceHostBroker.exe",
-                "?:\\Windows\\System32\\wbem\\WmiPrvSE.exe",
-                "?:\\Windows\\System32\\SrTasks.exe",
-                "?:\\Windows\\System32\\lsass.exe",
-                "?:\\Windows\\System32\\diskshadow.exe",
-                "?:\\Windows\\System32\\dfsrs.exe",
-                "?:\\Program Files\\*.exe",
-                "?:\\Program Files (x86)\\*.exe",
-                "?:\\WindowsAzure\\*\\WaAppAgent.exe",
-                "?:\\Windows\\System32\\vssadmin.exe",
-                "?:\\Windows\\VeeamVssSupport\\VeeamGuestHelper.exe",
-                "?:\\Windows\\System32\\dllhost.exe",
-                "?:\\Windows\\System32\\mmc.exe",
-                "?:\\Windows\\System32\\SettingSyncHost.exe",
-                "?:\\Windows\\ImmersiveControlPanel\\SystemSettings.exe",
-                "?:\\Windows\\System32\\SystemSettingsAdminFlows.exe",
-                "?:\\Windows\\Temp\\rubrik_vmware???\\snaptool.exe",
-                "?:\\Windows\\System32\\inetsrv\\w3wp.exe",
-                "?:\\$WINDOWS.~BT\\Sources\\*.exe",
-                "?:\\Windows\\System32\\wsmprovhost.exe",
-                "?:\\Windows\\System32\\spool\\drivers\\x64\\3\\x3jobt3?.exe",
-                "?:\\Windows\\System32\\mstsc.exe",
-                "?:\\Windows\\System32\\esentutl.exe",
-                "?:\\Windows\\System32\\RecoveryDrive.exe",
-                "?:\\Windows\\System32\\SystemPropertiesComputerName.exe") and
-
-  /* privileged local groups */
-  (group.name:("*admin*","RemoteDesktopUsers") or
-   winlog.event_data.TargetSid:("S-1-5-32-544","S-1-5-32-555"))
+host.os.type:windows and event.category:iam and event.action:user-member-enumerated and 
+  (
+    group.name:(*Admin* or "RemoteDesktopUsers") or
+    winlog.event_data.TargetSid:("S-1-5-32-544" or "S-1-5-32-555")
+  ) and 
+  not winlog.event_data.SubjectUserName: (*$ or "LOCAL SERVICE" or "NETWORK SERVICE")
```

Comment:
It will be painful to do it, but I think we should keep the exclusions

Reply:
++